protobuf/grpc_service.proto

```diff
@@ -1035,6 +1035,61 @@ message InferBatchStatistics
   StatisticDuration compute_output = 4;
 }
 
+//@@
+//@@.. cpp:var:: message InferResponseStat
+//@@
+//@@   Inference response statistics.
+//@@
+message InferResponseStat 
+{
+  //@@  .. cpp:var:: StatisticDuration compute_infer
+  //@@
+  //@@     The count and cumulative duration generate the inference response.
+  //@@
+  StatisticDuration compute_infer = 1;
+
+  //@@  .. cpp:var:: StatisticDuration compute_output
+  //@@
+  //@@     The count and cumulative duration to extract output tensor data
+  //@@     produced by the model framework / backend for the given response index.
+  //@@     For example, this duration should include the time to copy output
+  //@@     tensor data from the GPU.
+  //@@
+  StatisticDuration compute_output = 2;
+
+  //@@  .. cpp:var:: StatisticDuration success
+  //@@
+  //@@     Cumulative count and duration for successful inference
+  //@@     response.
+  //@@
+  StatisticDuration success = 3;
+
+  //@@  .. cpp:var:: StatisticDuration fail
+  //@@
+  //@@     Cumulative count and duration for failed inference
+  //@@     response.
+  //@@
+  StatisticDuration fail = 4;
+}
+
+//@@
+//@@.. cpp:var:: message InferResponseStatistics
+//@@
+//@@   Inference response statistics.
+//@@
+message InferResponseStatistics
+{
+  //@@
+  //@@.. cpp:var:: message InferResponseStat (repeated)
+  //@@
+  //@@   Cumalative statistics corresponding to responses generated by the
+  //@@   requests that create certain number of responses. For example, each
```

Comment:
Move the example part to L1170 and make the it more general like:

In other words, response statistics for requests that generate identical number of responses will be accumulated in a single InferResponseStatistics message.

Reply:
Updated.